test/functionalTest/cases/0908_update_duplicated_attributes/dup_attrs_avoid_db_error.test

```diff
@@ -129,16 +129,16 @@ Date: REGEX(.*)
 02. Update entity using the same attribute name twice
 =====================================================
 HTTP/1.1 200 OK
-Content-Length: 113
+Content-Length: 105
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
     "errorCode": {
-        "code": "422",
+        "code": "207",
```

Comment:
This shouldn't be 2xx as there is no partial success in this case for the update of an entity with duplicated attributes.

(NOTE: probably 422 is also wrong as according to "Error Response" rules at https://fiware.github.io/specifications/ngsiv2/stable this is a case in which the error is caused by request itself (i.e. they do not depend on the NGSIv2 server status) so 400 should be used instead. But this is a lateral issue)